### PR TITLE
[Chinese]Update the title of configure-upgrade-etcd.md

### PR DESCRIPTION
**Pull request by nianjiang**
[Chinese] Update the title of configure-upgrade-etcd.md

### CODE REVIEW

File: content/zh-cn/docs/tasks/administer-cluster/configure-upgrade-etcd.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: 为 Kubernetes 运行 etcd 集群
+title: 操作Kubernetes中的Etcd集群
```
Comment (windsonsea) [inline content/zh-cn/docs/tasks/administer-cluster/configure-upgrade-etcd.md:2]:
```suggestion
title: 操作 Kubernetes 中的 etcd 集群
```

**Comment by windsonsea** [comment]
Checked history, this file has no more in-synced content with the en upstream:

- [en history](https://github.com/kubernetes/website/commits/main/content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md)
- [zh history](https://github.com/kubernetes/website/commits/main/content/zh-cn/docs/tasks/administer-cluster/configure-upgrade-etcd.md)

File: content/zh-cn/docs/tasks/administer-cluster/configure-upgrade-etcd.md (hunk 1 above)
Comment (nianjiang) [reply content/zh-cn/docs/tasks/administer-cluster/configure-upgrade-etcd.md]: Done.

**Comment by windsonsea** [comment]
You need to squash your commits into a single one. Refer to: https://kubernetes.io/docs/contribute/new-content/open-a-pr/#squashing-commits

/hold until the commits are squashed.

**Comment by nianjiang** [comment]
Done (The 2 commits has been squashed).

**Comment by windsonsea** [comment]
Not yet: https://github.com/kubernetes/website/pull/43963/commits

**Comment by nianjiang** [comment]
> https://github.com/kubernetes/website/pull/43963/commits

The 2 commit are:
[[Chinese]Update the title of configure-upgrade-etcd.md](https://github.com/kubernetes/website/pull/43963/commits/a0a6cb7064aeaf628ed7860d8f6a56ea36ce80dc)
[Merge branch 'kubernetes:main' into patch-1](https://github.com/kubernetes/website/pull/43963/commits/2b8680fbf6fc78b8a20610aa38e342cdf09c3625)


About this commit: [Merge branch 'kubernetes:main' into patch-1](https://github.com/kubernetes/website/pull/43963/commits/2b8680fbf6fc78b8a20610aa38e342cdf09c3625)
***It is related to sync from 'kubernetes:main' into patch-1***

Do you need me to squashed that commit?

**Pull request merged**